tests/test_gw_repeater.sh

```diff
@@ -55,8 +55,8 @@ check_wsl() {
     # Docker fails to route traffic from the host to the containers
     # if the IP is anything but what was allocated by the daemon
     # when the container was created
```

Comment:
Is this comment still valid?

Reply:
To me it looks like it's no longer relevant, I removed it.